sdk-api/src/main/kotlin/io/etip/sdk/api/pokemon/factory/PokemonSdkFactory.kt

```diff
@@ -0,0 +1,15 @@
+package io.etip.sdk.api.pokemon.factory
+
+import io.etip.sdk.api.pokemon.config.PokemonClientConfig
+import io.etip.sdk.api.pokemon.integration.PokemonApiService
+import io.etip.sdk.api.pokemon.integration.impl.DefaultPokemonApiService
+import io.etip.sdk.core.BaseHttpClient
+import io.etip.sdk.core.HttpClientFactory
+
+object PokemonSdkFactory {
```

Comment:
Please introduce the concept of DI as well.

https://github.com/eTipio/Dayforce-Pandium/tree/develop/app/src/main/kotlin/com/dayforce/sdk/di

Use the service locator pattern